src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -566,8 +570,8 @@ const FileTreePane = () => {
     }, 2000);
   };
 
-  const onRenaming = (absolutePathToEntity: string) => {
-    dispatch(openRenameEntityModal(absolutePathToEntity));
+  const onRenaming = (absolutePathToEntity: string, os: string) => {
```

Comment:
could you rename `onRenaming` to `onRename` and do the same for `onDeleting` to `onDelete` ?

Reply:
Done.